.azuredevops/components/rocprofiler.yml

```diff
@@ -46,11 +62,68 @@ jobs:
   - template: ${{ variables.CI_TEMPLATE_PATH }}/steps/checkout.yml
     parameters:
       checkoutRepo: ${{ parameters.checkoutRepo }}
+# Manually download rocm-core and aqlprofile, hard-coded 6.1.0 versions
```

Comment:
Should I use a variable for the rocm-core and aqlprofile versions? I haven't yet tried using other versions of these dependencies.

Reply:
This is fine for now to get live builds enabled. We have to refactor how aqlprofile gets incorporated in the future anyway. As for rocm-core, you can merge this solution and then next investigate what that debian package actually installs and see if we can get one of our existing pipelines to replicate producing those files.